requirements/base.txt

```diff
@@ -0,0 +1,135 @@
+cffi==1.14.1              # via cryptography
+chardet==3.0.4            # via aiohttp, requests, tabulator
+click==7.1.2              # via apache-superset, flask, flask-appbuilder, tableschema, tabulator
+colorama==0.4.3           # via apache-superset, flask-appbuilder
+contextlib2==0.6.0.post1  # via apache-superset
+croniter==0.3.34          # via apache-superset
+cryptography==3.0         # via apache-superset
+dataclasses==0.6          # via apache-superset
+decorator==4.4.2          # via retry
+defusedxml==0.6.0         # via python3-openid
+dnspython==2.0.0          # via email-validator
+docutils==0.15.2          # via botocore
+email-validator==1.1.1    # via flask-appbuilder
+et-xmlfile==1.0.1         # via openpyxl
+flask-appbuilder==3.0.1   # via apache-superset
+flask-babel==1.0.0        # via flask-appbuilder
+flask-caching==1.9.0      # via apache-superset
+flask-compress==1.5.0     # via apache-superset
+flask-cors==3.0.8         # via apache-superset
+flask-jwt-extended==3.24.1  # via flask-appbuilder
+flask-login==0.4.1        # via flask-appbuilder
+flask-migrate==2.5.3      # via apache-superset
+flask-openid==1.2.5       # via flask-appbuilder
+flask-sqlalchemy==2.4.4   # via flask-appbuilder, flask-migrate
+flask-talisman==0.7.0     # via apache-superset
+flask-wtf==0.14.3         # via apache-superset, flask-appbuilder
+flask==1.1.2              # via apache-superset, flask-appbuilder, flask-babel, flask-caching, flask-compress, flask-cors, flask-jwt-extended, flask-login, flask-migrate, flask-openid, flask-sqlalchemy, flask-wtf
+future==0.18.2            # via pyhive
+geographiclib==1.50       # via geopy
+geopy==2.0.0              # via apache-superset
+gunicorn==20.0.4          # via apache-superset
+humanize==2.5.0           # via apache-superset
+idna-ssl==1.1.0           # via aiohttp
+idna==2.10                # via email-validator, idna-ssl, requests, yarl
+ijson==3.1.post0          # via tabulator
+importlib-metadata==1.7.0  # via jsonschema, kombu, markdown
+isodate==0.6.0            # via apache-superset, tableschema
+itsdangerous==1.1.0       # via flask, flask-wtf
+jdcal==1.4.1              # via openpyxl
+jinja2==2.11.2            # via flask, flask-babel
+jmespath==0.10.0          # via boto3, botocore
+jsonlines==1.2.0          # via tabulator
+jsonschema==3.2.0         # via flask-appbuilder, tableschema
+kombu==4.6.11             # via celery
+linear-tsv==1.1.0         # via tabulator
+mako==1.1.3               # via alembic
+markdown==3.2.2           # via apache-superset
+markupsafe==1.1.1         # via jinja2, mako, wtforms
+marshmallow-enum==1.5.1   # via flask-appbuilder
+marshmallow-sqlalchemy==0.23.1  # via flask-appbuilder
+marshmallow==3.7.1        # via flask-appbuilder, marshmallow-enum, marshmallow-sqlalchemy
+msgpack==1.0.0            # via apache-superset
+multidict==4.7.6          # via aiohttp, yarl
+mysqlclient==1.4.2.post1  # via apache-superset
```

Comment:
`mysqlclient` is GPL licensed, and can't be included in the base requirements. It seems the new file structure is not working with FOSSA license checks, which we also need to resolve. cc @villebro @craig-rueda @dpgaspar for the 0.38 release

Reply:
@robdiciuccio et al. I wonder if we should switch to using the pure Python [PyMySQL](https://github.com/PyMySQL/PyMySQL) driver (potentially works better with `gevent`) which is under the MIT license.

Reply:
Note @robdiciuccio `mysqlclient` and `psycopg2` originate from `setup.py`.